content/applications/sales/crm/pipeline/manage_sales_teams.rst

```diff
@@ -0,0 +1,106 @@
+==================
+Manage sales teams
+==================
+
+The *Sales Teams* feature within Odoo's *CRM* app allows for the creation and management of multiple
+sales teams, each with their own assignment rules, invoicing targets, and roster of salespeople.
+
+Create a sales team
+===================
+
+To create a new sales team, go to :menuselection:`CRM app --> Configuration --> Sales Teams`, then
+click :guilabel:`New`.
+
+On the blank sales team form, enter a name in the :guilabel:`Sales Team` field.
+
+Next, select a :guilabel:`Team Leader` from the drop-down list.
+
+Set an :guilabel:`Email Alias` to automatically generate a lead/opportunity for this sales team
+whenever a message is sent to that unique email address. Choose whether to accept emails from
+:guilabel:`Everyone`, :guilabel:`Authenticated Partners`, :guilabel:`Followers Only`, or
+:guilabel:`Authenticated Employees`.
+
+Select a :guilabel:`Company` from the drop-down to assign this team to. *This field is only visible
+in multi-company databases, and is not required.*
```

Comment:
Suggestion to move this to a note, and reiterate the field name so there is no confusion :)
```suggestion
Select a :guilabel:`Company` from the drop-down to assign this team to.

.. note::
   The :guilabel:`Company` field is only visible in multi-company databases, and is not required.
```